### ISSUE TITLE
Pak channels?

### ISSUE THREAD

**Issue by danishkhan91** [snake24564/freechannels#1]
Good work bro on the hindi channels. U have pak CHannels?

**Comment by snake24564** [comment]
No.  But if you give me a list of some channels you would like I can try my best to get them.

**Comment by ss0878** [comment]
Seems channels are down? Do you have any telegram group?

**Comment by snake24564** [comment]
Fixed.  And no I don’t have telegram

**Comment by ss0878** [comment]
I appreciate the fix. I'm not sure for how long you will keep this up or try to update the channels. But I would like to help by adding icons to the channels. If possible?

**Comment by snake24564** [comment]
I already have channel logos for 95% of the channels.
The channels would be updated after I add a few more channels.
I don't plan on taking this down or changing the parameters of the authentication or anything.
The next update will include more channels with channel logos.

**Comment by danishkhan91** [comment]
Hey brother. I have an old white Shava Mx Iptv box. Is there a way to customize the box with a different iptv firmware?

**Comment by snake24564** [comment]
If it runs android you can download vlc and play m3u playlists off it... Also there is an app on Android called STB EMU which is a emulator to run iptv services.  You can download STB EMU from https://stbemu-pro.en.aptoide.com/app

**Comment by danishkhan91** [comment]
Thank you.
I will check it out

**Comment by ss0878** [comment]
Perfect. Have you thought of anything for on demand stuff like movies and shows?

**Comment by danishkhan91** [comment]
No. For subscription?

**Comment by ss0878** [comment]
Sorry, @danishkhan91 . I was trying to ask @snake24564

**Comment by snake24564** [comment]
> Sorry, @danishkhan91 . I was trying to ask @snake24564

Hey.  I don't have any plan for VOD content yet but probably in the future.  And I have a Skype.  My Skype ID is snake24564
